### PR TITLE
Add covid reference notebook

### PR DESCRIPTION
**Pull request by mxposed**
This is a copy of the notebook from here https://github.com/NUPulmonary/scarches-covid-reference
just to put them in one place, as suggested by @M0hammadL 

Please, do not merge yet, as I need to rerun it, because we changed the annotations on GEO.

This is the basic structure, with the 2 additional files with utility functions. Please let me know if the structure is ok, or if I need to modify anything.

Thank you

### CODE REVIEW

**Comment by M0hammadL** [comment]
Hi Nik, thanks a lot. Unfirtatenly we have updated the whole repo with the new version. We now use PyTorch instead of Tensorflow. I guess you can rerun the whole analysis with scANVI mode. You can use create a reference atlas and then project and classify it without using any knn.

**Comment by mxposed** [comment]
Hey. That's good news! Could you please tell me what changes practically for the package and for this pull request?

Do I get it right that this is the relevant notebook now? https://github.com/theislab/scarches/blob/master/notebooks/scanvi_surgery_pipeline.ipynb

I can adapt this covid notebook to that one, but it'll take some time. Please let me know what's the best next step?

**Comment by M0hammadL** [comment]
Yes, that is the writing notebook:

I would perform this analysis: 
1- train scANVI reference model with  your reference data
2- treat your query data as unlabeled (see notebook) 
3- transfer cell type labels from reference to query (you don't need any knn claassifier the model does it for you)

**Comment by M0hammadL** [comment]
> Hey. That's good news! Could you please tell me what changes practically for the package and for this pull request?
> 
The package now uses PyTorch library instead of TensorFlow therefore you need to install the newest version first. Since the library and the implementations have changed you need to rerun your analysis using the new version. 
> Do I get it right that this is the relevant notebook now? https://github.com/theislab/scarches/blob/master/notebooks/scanvi_surgery_pipeline.ipynb

Yes, that is the right notebook:

I would perform this analysis:
1- train scANVI reference model with your reference data
2- treat your query data as unlabeled (see notebook)
3- transfer cell type labels from reference to query (you don't need any knn classifier the model does it for you)
> 
> I can adapt this covid notebook to that one, but it'll take some time. Please let me know what's the best next step?

If you manage to rerun the same analysis with the new version you can ask for another pull request.

Please let me know if you need any other help.

Mo

**Comment by mxposed** [comment]
Thank you very much for the explanations. Ok, I'll close this for now and will open the new one when I have a new version
